cookiecutter-django-ida/{{cookiecutter.repo_name}}/Makefile

```diff
@@ -184,3 +184,20 @@ db-shell: # Run the app shell as root, enter the app's database
 
 %-attach:
 	docker attach {{cookiecutter.project_name}}.$*
+
+github_docker_build:
+	docker build . -f Dockerfile --target app -t openedx/repo_name
+	docker build . -f Dockerfile --target newrelic -t openedx/repo_name:latest-newrelic
+
+github_docker_tag: docker_build
+	docker tag openedx/repo_name openedx/repo_name:${GITHUB_SHA}
+	docker tag openedx/repo_name:latest-newrelic openedx/repo_name:${GITHUB_SHA}-newrelic
+
+github_docker_auth:
+	echo "$$DOCKERHUB_PASSWORD" | docker login -u "$$DOCKERHUB_USERNAME" --password-stdin
+
+github_docker_push: docker_tag docker_auth ## push to docker hub
```

Comment:
docker tag should be replaced by github_docker_tag and docker auth with github_docker_auth

Reply:
Good catch, fixed.